### PR TITLE
Instance group and notifications details popover

### LINKED ISSUE #11871: Add help text popovers to /#/notifications details fields
##### ISSUE TYPE
 - Feature Idea

##### SUMMARY
Spawned out of https://github.com/ansible/awx/issues/9215

See /#/settings for how this should work.  If you look the details view for a particular settings group you'll see help text popovers next to many (if not all) of the fields.  The help text matches up with the fields in the form.  We want to bring all the forms/details to parity in this regard.  If you see form fields that are missing help text, feel free to add it.

Here's a list of details views and fields that have help text in the form but not in the details:

**Notification**

- Recipient list
- Timeout
- Grafana URL
- Tags for the annotation
- Destination channels or users
- Destination channels
- Notification color
- Destination SMS number(s)
- HTTP Headers

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
Resolves #11871 and #11872
##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - UI

##### AWX VERSION

### CODE REVIEW

**Review by kialam — APPROVED** [review]

File: awx/ui/src/screens/NotificationTemplate/shared/Notifications.helptext.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { t } from '@lingui/macro';
+
+const NotificationHelpText = () => ({
```
Comment (keithjgrant) [inline awx/ui/src/screens/NotificationTemplate/shared/Notifications.helptext.js]: why a function instead of exporting the values in an object?

Comment (AlexSCorey) [reply awx/ui/src/screens/NotificationTemplate/shared/Notifications.helptext.js]: Good catch....habitt

**Review by keithjgrant — APPROVED** [review]
👍🏻

**Comment by jay-steurer** [comment]
Tested and approved.

**Review by InnocentK — APPROVED** [review]
See @jay-steurer's comment above. 

Also I started the E2E tests and those results should be available later today

**Comment by InnocentK** [comment]
The above failed test is unrelated to these changes so its fine to merge

**Pull request merged**